tokio/src/sync/mpsc/list.rs

```diff
@@ -35,8 +35,13 @@ pub(crate) enum TryPopResult<T> {
     /// Successfully popped a value.
     Ok(T),
     /// The channel is empty.
+    ///
+    /// Returned after `Rx::close()` when no values remain,
+    /// even if senders still exist.
```

Comment:
I suggest this wording:
```suggestion
    /// Note that `list.rs` only tracks the close state set by senders. If the
    /// channel is closed by `Rx::close()`, then `TryPopResult::Empty` is still
    /// returned, and the close state needs to be handled by `chan.rs`.
```